src/SelectField/SelectField.jsx

```diff
@@ -161,70 +168,172 @@ const SelectField = React.createClass({
     };
   },
 
+  componentDidMount() {
+    if (this.props.open) {
+      /* eslint-disable */
```

Comment:
Small note: use rule specific comments for disabling, eg:

`/* eslint react/no-did-mount-set-state: 0 */`

Also, why do we need to use `setState()` in `componentDidMount`?